src/components/Node/Node.module.scss

```
@@ -60,4 +64,8 @@
     border-width: 1px;
     border-style: solid;
   }
+
+  &.childDeclined {
+    border: 1px solid #404040
```

Use `$black-medium` here too
```suggestion
    border: 1px solid $black-medium
```